packages/text-area/test/text-area.test.js

```diff
@@ -0,0 +1,372 @@
+      }
+
+      ['placeholder', 'value'].forEach((prop) => {
+        it('should set string property ' + prop, () => {
+          assertPropCanBeSet(prop, 'foo');
+        });
+      });
+
+      ['disabled'].forEach((prop) => {
+        it('should set boolean property ' + prop, () => {
+          assertPropCanBeSet(prop, true);
+        });
+      });
+
+      ['maxlength', 'minlength'].forEach((prop) => {
+        it('should set numeric attribute ' + prop, () => {
+          assertAttrCanBeSet(prop, 2);
+        });
+      });
+
+      ['autocomplete'].forEach((prop) => {
+        it('should set boolean attribute ' + prop, () => {
+          assertAttrCanBeSet(prop, 'on');
+        });
+      });
+
+      ['autocapitalize'].forEach((prop) => {
+        it('should set boolean attribute ' + prop, () => {
+          assertAttrCanBeSet(prop, 'none');
+        });
+      });
+
+      ['autocomplete', 'autocorrect', 'readonly', 'required'].forEach((prop) => {
+        it('should set boolean attribute ' + prop, () => {
+          assertAttrCanBeSet(prop, true);
+          assertAttrCanBeSet(prop, false);
+        });
+      });
+
+      describe('input field', () => {
+        let inputField;
+
+        beforeEach(() => {
+          inputField = textArea.shadowRoot.querySelector('[part="input-field"]');
+        });
+
+        it('should propagate invalid property to the input container', () => {
+          textArea.invalid = true;
+          expect(inputField.invalid).to.be.true;
+
+          textArea.invalid = false;
+          expect(inputField.invalid).to.be.false;
+        });
+
+        it('should propagate readonly property to the input container', () => {
+          textArea.readonly = true;
+          expect(inputField.readonly).to.be.true;
+
+          textArea.readonly = false;
+          expect(inputField.readonly).to.be.false;
+        });
+
+        it('should propagate disabled property to the input container', () => {
+          textArea.disabled = true;
+          expect(inputField.disabled).to.be.true;
+
+          textArea.disabled = false;
+          expect(inputField.disabled).to.be.false;
+        });
+
+        it('should propagate theme attribute to the input container', () => {
+          textArea.setAttribute('theme', 'align-center');
+          expect(inputField.getAttribute('theme')).to.equal('align-center');
+        });
+      });
+    });
+
+    describe('binding', () => {
+      it('default value should be empty string', () => {
+        expect(textArea.value).to.be.equal('');
+      });
+
+      it('setting textarea value updates value', () => {
+        native.value = 'foo';
+        native.dispatchEvent(new Event('input', { bubbles: true, cancelable: true, composed: true }));
+        expect(textArea.value).to.be.equal('foo');
+      });
+
+      it('setting native value updates has-value attribute', () => {
+        textArea.value = 'foo';
+        expect(textArea.hasAttribute('has-value')).to.be.true;
+      });
```

Comment:
This is not setting the native value. May be that's intended, not sure if we need tests to check that inputting into the native element updates has-value.

Reply:
Updated all the test descriptions in this suite.